dist/src/main/resources/mc_locale/mc_locale_java-source_en.properties

```diff
@@ -59,6 +59,7 @@ PHANT_IO=Sparkfun [phant.io]
 PI4J=pi4j
 REGISTERED=Registered
 S_AIR_QUALITY=Air quality
+S_HUE_LIGHT=Hue light
```

Comment:
@jkandasa this is for variables mapper define with (Light level, RGB, Status). Can you confirm the suppression?